core/blockchain/process.go

```diff
@@ -526,7 +523,7 @@ func (b *BlockChain) disconnectBlock(ib meerdag.IBlock, block *types.SerializedB
 		for _, stxo := range stxos {
 			pkss = append(pkss, stxo.PkScript)
 		}
-		err := b.indexManager.DisconnectBlock(block, pkss, ib, vmbid)
+		err := b.indexManager.DisconnectBlock(block, pkss, ib, 0)
```

Comment:
why `0` is hardcoded?  looks the `blocknumber` is returned from `b.VMService().DisconnectBlock(block)`

Reply:
> why `0` is hardcoded? looks the `blocknumber` is returned from `b.VMService().DisconnectBlock(block)`

This needs to be discard temporarily